src/components/About.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import AboutImg from '../assets/about-image.svg'
+
+function About() {
+  return (
+    <div className='w-full my-32 ' > 
+        <div className=' flex justify-center flex-col bg-lightBlue'>
```

Comment:
I think you can add and try:
1.  w-1/2  sm:w-full (just try I am not sure) 
2. h2 text color can be: text-mediumBlue
3. paragraph size can be smaller: 2xl or xl ? 
4. for button style: rounded-full  text-[#FFFFFF] flex justify-center